backend/framework/qlf/dashboard/bokeh/qacountpix/main.py

```diff
@@ -4,14 +4,16 @@
 from bokeh.models.widgets import PreText, Div
 from bokeh.models import HoverTool, ColumnDataSource, PrintfTickFormatter
 from bokeh.models import LinearColorMapper, ColorBar
+from dashboard.bokeh.qlf_plot import html_table
 
 import numpy as np
 
 from dashboard.bokeh.helper import write_description, write_info, \
     get_merged_qa_scalar_metrics
 
 from dashboard.bokeh.helper import get_palette
-from dashboard.bokeh.qlf_plot import alert_table, mtable
+from dashboard.bokeh.qlf_plot import alert_table, mtable, \
```

Comment:
This table doesn't look right. Is this PR supposed fixes it? 

![screenshot 2018-10-18 13 33 19](https://user-images.githubusercontent.com/13886329/47169625-64c2ed80-d2da-11e8-99fd-1cc181bee7f4.png)

It's not really clear what this PR is for.

Reply:
This PR was created from master, before the fix_tables. It correct the AMP plots (decimals, color scale, reference values etc).

Reply:
ok